tasks/rke2.yml

```diff
@@ -231,16 +231,6 @@
       changed_when: false
       when: not rke2_airgap_mode
 
-- name: Copy Custom Manifests
-  ansible.builtin.template:
-    src: "{{ item }}"
-    dest: "{{ rke2_data_path }}/server/manifests/"
-    owner: root
-    group: root
-    mode: 0644
-  with_items: "{{ rke2_custom_manifests }}"
-  when: rke2_custom_manifests
-
```

Comment:
I think we should keep this Task and rather replace the following Block with

```yaml
- name: Copy Static Pods
  ansible.builtin.copy:
    src: "{{ item }}"
    dest: "{{ rke2_data_path }}/agent/pod-manifests/"
    owner: root
    group: root
    mode: 0644
  with_items: "{{ rke2_static_pods }}"
  when: rke2_static_pods
```
 to keep the original behavior.
Basically we need to revert this change https://github.com/lablabs/ansible-role-rke2/pull/260/files#diff-18777598e84dcc9ba07f83aaee64bfcdf001257e84ed46f349bd1a84a178dbebR239-R266

Reply:
Either way is fine with me. Feel free to close it.